tests/legacy/ApiTest.php

```diff
@@ -23,7 +23,6 @@
 namespace Friendica\Test\legacy;
```

Comment:
In fact, after removing `include/api.php` at all, this class should be deleted as well.
And all the current methods should be either
- moved to the right class
- removed if it's content is deprecated
- transformed in case the "old" test is still valid, but needs refactoring because of the new paradigm